__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require('../app');
+const connection = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    if (connection) connection.end();
+})
+
+describe('GET /api/topics', () => {
+    it('status 200, responds with an array of topic objects,  each of which should have the following properties: slug, description.', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
+                const {topics} = body;
+                const {topicData} = testData;
+
+                expect(topics).toBeArray();
+                expect(topics.length).toBe(topicData.length);
```

Comment:
We don't need to compare the length of two datasets here - you can specify the expected length of the test data i.e. there are 3 topics to expect length of 3.